src/state/selection/logics.ts

```diff
@@ -32,10 +32,10 @@ const storeColorsLogic = createLogic({
             }
             return newAgent;
         });
-        dispatch(receiveAgentNamesAndStates(newUiData));
+        dispatch(setDefaultUIData(newUiData));
```

Comment:
I'm confused by this naming: why are we setting default data when the `APPLY_USER_COLOR_SELECTION` action has been triggered?

Reply:
This is slightly confusing as it is somewhat "in between" the old way and the new way.

In this PR where I am not making functional changes, there is only one piece of UI Display Data, what is currently called `agentUiNames` and I propose to call `defaultUiData` in the future. Here in the current paradigm it gets modified when users make color selections. The names only really make sense in the context of the next PR. I'm just pulling these changes out to make that more readable, if its more confusing I can just point the other PR at main.